configs/RRIOT_F722/config.h

```diff
@@ -99,5 +99,4 @@
 
 #define BEEPER_INVERTED
 
-#define PINIO1_PIN PB0
-#define PINIO1_BOX 39
+#define PINIO1_BOX 40
```

Comment:
This will not work without assigning a resource

Reply:
This config allows us to power VTX by default. If users want to control it, they will need to use the CLI to assign the resource (we are providing instructions). We are doing this so that we do not need to solder the VTX jumper for the default setting we want. We have tested this and it works the way we need it to.